cli/src/main/java/de/jplag/cli/options/CliOptions.java

```diff
@@ -54,8 +54,8 @@ public class CliOptions implements Runnable {
     public String resultFile = "results";
 
     @Option(names = {"-M",
-            "--mode"}, description = "The mode of JPlag. One of: ${COMPLETION-CANDIDATES} (default: ${DEFAULT_VALUE}). If VIEW is chosen, you can optionally specify a path to an existing report.")
-    public JPlagMode mode = JPlagMode.RUN_AND_VIEW;
+            "--mode"}, description = "The mode of JPlag. By default JPlag will automatically select the mode based on the given input files. If none are given the report viewer will open on the file upload page. If a single result zip is given it will be opened in the report viewer directly. Otherwise, JPlag will check the given submissions and show the result in the report viewer. One of: ${COMPLETION-CANDIDATES} (default: ${DEFAULT_VALUE})")
+    public JPlagMode mode = JPlagMode.AUTO;
```

Comment:
What was the consensus here from the last meeting? Less details, right? @jplag/hiwi @jplag/maintainer